### PR TITLE
feat: create signup screen

### LINKED ISSUE #3: SignUp Design
## DESCRIPTION

We implement the design of the signup of our app.
## DESIGN

image

### PR DESCRIPTION
**Pull request by Alfredo20164920**
# Description

What did you implemented/Why did you implemented this:

- Create the code following the figma´s desing (only the structure).
- Add redirection to sign in if the user have an account

## Screenshots

image

### CODE REVIEW

File: src/screens/Login/Signup/SignUpScreen.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        <Input label={'Email Address'} placeholder={'john@doe.com'} />
+        <Input
+          label={'Password'}
+          secure={true}
+          placeholder={'min. 8 characters'}
+        />
+
+        <Pressable
+          onPress={() => {
+            console.log('hi');
```
Comment (BrandonVG) [inline src/screens/Login/Signup/SignUpScreen.jsx:34]: Remember to delete console.log lines

File: src/screens/Login/Signup/SignUpScreen.jsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+              Register
+            </Text>
+          )}
+        </Pressable>
+
+        <SeparationComponent data={'Or'} />
+
+        <Pressable
+          onPress={() => {
+            console.log('hi');
```
Comment (BrandonVG) [inline src/screens/Login/Signup/SignUpScreen.jsx:58]: Remember to delete console.log lines

**Review by BrandonVG — APPROVED** [review]
Nice job pretty design
Like gato

**Review by Lizzflores — APPROVED** [review]

**Review by AMCernas — APPROVED** [review]
Good job!

**Pull request merged**